### PR TITLE
Added features section

### PR DESCRIPTION
**Pull request by JeremyTheocharis**
Still as a draft until the analytics feature is finished. Feedback would be great.

Added @Scarjit and @DaruZero as reviewers, as these articles provide a different perspective on our documentation and show where there is stuff missing (e.g., where do we have the historian functionalitiies like min, max, downsampling, etc. documented).

### CODE REVIEW

**Comment by DaruZero** [comment]
Maybe consider writing a "features" archetype? so that in the future if we want to add more pages we have a skel for it

File: umh.docs.umh.app/content/en/docs/features/unified-namespace/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Configure your IoT devices and devices on the shopfloor to use the in-built MQTT broker of the United Manufacturing Hub by specifying the MQTT protocol, selecting unencrypted (1883) / encrypted (8883) ports depending on your configuration, and send the messages into a topic starting with `ia/raw`. From there on, you can start processing the messages in Node-RED by reading in the messages again via MQTT or Kafka, adjusting the payload or the topic to match the [UMH datamodel](/docs/architecture/datamodel/) and sending it back again to MQTT or Kafka.
+
+If you send the messages into other topics, some features might not work correctly (see also [limitations](#what-are-the-limitations)).
+
+{{% notice tip %}}
+**Recommendation:** Send messages from IoT devices via MQTT and then work in Kafka only.
+{{% /notice %}} 
+
+## What are the limitations?
```
Comment (Scarjit) [inline umh.docs.umh.app/content/en/docs/features/unified-namespace/index.md:44]: Max 249 chars (https://github.com/apache/kafka/commit/ad3dfc6ab25c3f80d2425e24e72ae732b850dc60)
Legal char regex: [a-zA-Z0-9\\._\\-]

Comment (Scarjit) [reply umh.docs.umh.app/content/en/docs/features/unified-namespace/index.md:44]: Max message size defaults to 1000000 byte (https://docs.confluent.io/4.0.0/clients/librdkafka/CONFIGURATION_8md.html)

File: umh.docs.umh.app/content/en/docs/features/unified-namespace/index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- Easy integration using MQTT: Many modern shopfloor equipment can send and receive data using the MQTT protocol. 
+- Easy integration with legacy equipment: Using tools like [Node-RED](/docs/architecture/microservices/core/node-red/), data can be easily extracted from various protocols such as Siemens S7, OPC-UA, or Modbus
+- Get notified in real-time via MQTT: The Unified Namespace allows you to receive real-time notifications via MQTT when new messages are published. This can be useful for applications that require near real-time processing of data, such as an AGV waiting for new commands.
+- Retrieve past messages from Kafka logs: By looking into the Kafka logs, you can always be aware of the last messages that have been sent to a topic. This allows you to replay certain scenarios for troubleshooting or testing purposes.
+- Efficiently process messages from millions of devices: The Unified Namespace is designed to handle messages from millions of devices in your factory, even over unreliable connections. By using Kafka, you can efficiently at-least-once process each message, ensuring that no data is lost or duplicated.
```
Comment (Scarjit) [inline umh.docs.umh.app/content/en/docs/features/unified-namespace/index.md:29]: "at-least-once" -> 1 or more times, not "no data is lost or duplicated."

**Review by Scarjit — CHANGES_REQUESTED** [review]

**Review by Scarjit — APPROVED** [review]

**Pull request merged**